.github/actions/setup-zeebe/action.yml

```diff
@@ -0,0 +1,61 @@
+# This action sets up the environment with the required tech stack in order to build, install, and
+# run Zeebe.
+
+---
+name: Setup Zeebe
+description: Sets up the required stack to build, install, and run Zeebe
+
+inputs:
+  go:
+    description: If true, will set up Golang; defaults to true
+    required: false
+    default: "true"
+  java:
+    description: If true, will set up Java; defaults to true
+    required: false
+    default: "true"
+  java-version:
+    description: The JDK version to setup
+    default: "17"
```

Comment:
:question:  Not sure, but was 8.0 still with JDK 11?

Reply:
Ah, good idea to double check! Luckily 8.0 uses 17: https://github.com/camunda/zeebe/blob/stable/8.0/parent/pom.xml#L29